src/main/java/study/step3/domain/Car.java

```diff
@@ -0,0 +1,24 @@
+package study.step3.domain;
+
+import study.step3.domain.strategy.MoveStrategy;
+import study.step3.domain.strategy.RandomMoveStrategy;
+
+public class Car {
+    private int position;
+
+    private final MoveStrategy moveStrategy;
+
+    public Car() {
+        this.moveStrategy = new RandomMoveStrategy();
+    }
```

Comment:
```suggestion
    public Car(MoveStrategy moveStrategy) {
        this.moveStrategy = moveStrategy;
    }
```
기본 생성자 외에 위와 같은 생성자를 추가하면 테스트가 가능해 지지 않을까?